internal/evaluator/conftest_evaluator.go

```diff
@@ -143,34 +146,39 @@ func createConfigJSON(fs afero.Fs, dataDir string, p *policy.Policy) error {
 		Exclude     *[]string `json:"exclude,omitempty"`
 		Include     *[]string `json:"include,omitempty"`
 		Collections *[]string `json:"collections,omitempty"`
+		WhenNs      int64     `json:"when_ns"`
 	}
 	pc := &policyConfig{}
 
 	// TODO: Once the NonBlocking field has been removed, update to dump the spec.Config into an updated policyConfig struct
 	if p.Exceptions != nil {
-		log.Debug("Non-blocking exceptions found. These will be written to file", dataDir)
+		log.Debug("Non-blocking exceptions found. These will be written to file ", dataDir)
 		pc.NonBlocking = &p.Exceptions.NonBlocking
 	}
 	if p.Configuration != nil {
-		log.Debug("Include rules found. These will be written to file", dataDir)
+		log.Debug("Include rules found. These will be written to file ", dataDir)
 		if p.Configuration.Include != nil {
 			pc.Include = &p.Configuration.Include
 		}
-		log.Debug("Exclude rules found. These will be written to file", dataDir)
+		log.Debug("Exclude rules found. These will be written to file ", dataDir)
 		if p.Configuration.Exclude != nil {
 			pc.Exclude = &p.Configuration.Exclude
 		}
-		log.Debug("Collections found. These will be written to file", dataDir)
+		log.Debug("Collections found. These will be written to file ", dataDir)
 		if p.Configuration.Collections != nil {
 			pc.Collections = &p.Configuration.Collections
 		}
 	}
-	// Check to see that we've actually added any values to the policyConfig struct.
-	// If so, we'll update the config map. Otherwise, this is skipped.
-	if (policyConfig{} != *pc) {
-		config["config"] = map[string]interface{}{
-			"policy": pc,
-		}
+
+	// Now that the future deny logic is handled in the ec-cli and not in rego,
```

Comment:
This. I really like the fact that the time we're evaluating against does not change between each image being validated.

Reply:
+1, I can imagine some potential weirdness if one conftest invocation ran a millisecond before midnight and the next ran a millisecond after.